clients/cli/src/main/java/org/apache/gravitino/cli/commands/TableAudit.java

```diff
@@ -53,10 +57,22 @@ public void handle() {
     try {
       NameIdentifier name = NameIdentifier.of(schema, table);
       gTable = tableCatalog().loadTable(name);
+
+    } catch (NoSuchMetalakeException err) {
+      exitWithError(ErrorMessages.UNKNOWN_METALAKE);
+    } catch (NoSuchCatalogException err) {
+      exitWithError(ErrorMessages.UNKNOWN_CATALOG);
+    } catch (NoSuchSchemaException err) {
+      exitWithError(ErrorMessages.UNKNOWN_SCHEMA);
     } catch (Exception exp) {
       exitWithError(exp.getMessage());
     }
 
+    if (gTable == null) {
+      exitWithError("Failed to retrieve table details.");
+      return;
```

Comment:
I don't have a strong opinion about this additional `return` line, though I'm more leaning towards having it (no harm + code readability).
However, when we decide to add these `return`s, we need to be consistent.
For example, line 68 doesn't have a following `return` statement, this would confuse others.